### ISSUE TITLE
Texture Bugs

### ISSUE THREAD

**Issue by feedbackex** [travnick/CryBlend#182]
Hi guys,

Texture export not more working correctly.
__mtl__ files just have texture names, not filepath.
Also cycles mode is not more working anymore.

Old system used:
```
        image_path = utils.get_relative_path(image_path,
                                             self.__textures_parent_directory)
```
New system use: 
```
image_path = utils.trim_path_to(dds_path, "Objects")
```
function that called from __export_library_image function in export.py.

@dwhite4  could you check it?

**Comment by ochounos** [comment]
I was checking about cycles. it doesn't work for me either. I found that in utils.py line 477 `if texture_slot and texture_slot.texture.type == 'IMAGE':` returns type NONE so the image is not included. I'm testing with blender 2.77
I will try to find the problem and I will report here again.

**Comment by ochounos** [comment]
I have added a commit to fix in cycles. I don't know how to test it in blender render. Also didn't test extensively other image paths. Please, test and provide more info or one example where it's not working to reproduce the error.

**Comment by feedbackex** [comment]
Thank you @ochounos, cycles render now is working like blender render.
But main problem still occur. Cryblend just export texture name not full path.

**Comment by ochounos** [comment]
Let me try more paths to try to reproduce the problem.

**Comment by feedbackex** [comment]
@ochounos, thank you for your attention. But last code does not works for me. Also preceding code works just for textures that located under Objects directory. So do not works if texture located directly under Textures directory.

If I want to export right texture path I must know a root directory about my game. Blender have a select texture directory tool for that.

First of all there is a logical problem about that tool. Cryengine works based game directory system. So main directory must start with game folder via that user can create a lot of games, and folders for that games with different names.

Finally, be consistency, I decided to change __Select Texture Directory__ to __Select Game Directory__
and all texture paths are relative that game directory variable which named as game_dir.

And all texture paths now are working, just needed you must select game directory.

**Comment by ochounos** [comment]
This issue has been already fixed in #188. Reopen it if I'm wrong.
